src/platform/forms-system/src/js/web-component-patterns/titlePattern.js

```diff
@@ -34,7 +39,12 @@ export const Title = ({ title, description }) => (
  *
  * @returns {UISchemaOptions}
  */
-export const titleUI = (title, description) => {
+export const titleUI = (
+  title,
+  description,
+  headerLevel = 3,
```

Comment:
let's sync up on this, I think I know a way we can just pass a single object, which would be preferred, since it will match all of our other patterns.